picard/coverart/providers/__init__.py

```diff
@@ -62,37 +63,52 @@ def __init__(self, parent=None):
 def register_cover_art_provider(provider):
     _cover_art_providers.register(provider.__module__, provider)
     if hasattr(provider, 'OPTIONS') and provider.OPTIONS:
-        provider.OPTIONS.NAME = provider.NAME
-        provider.OPTIONS.TITLE = provider.TITLE or provider.NAME
+        provider.OPTIONS.NAME = provider.name
+        provider.OPTIONS.TITLE = provider.title
```

Comment:
Why do we actually need `provider.OPTIONS`? I checked the existing plugins, and they don't use this but register an options page separately. Not sure what advantage using provider.OPTIONS would bring.

Reply:
For test I tried to use `OPTIONS` for the Fanart.tv plugin. I think using `ProviderOptions` provides very little benefit. Basically you replace a custom three line constructor necessary to set `self.ui` and calling `self.ui.setupUi()` with a single class property `_options_ui`, but that's something we could even do in the base `OptionsPage` class. Apart from that using `ProviderOptions` only spares you a single call to `register_options_page` and you don't have to specify a parent. The visible effect for the user is that all cover art options show up under the cover art section and not also under plugins, but that's something we could easily change in the plugins (and I already though about doing this).

Reply:
This is the diff for Fanarttv: https://gist.github.com/phw/08ba5a69d076a6c89c219da3821d1274

Reply:
I think your diff is reversed, but i get the point and i agree we can get rid of this  and directly use OPTIONS. It has to be in a different PR though IMHO.